helm_deploy/hmpps-tier/values.yaml

```diff
@@ -26,12 +26,6 @@ generic-service:
     annotations:
       nginx.ingress.kubernetes.io/custom-http-errors: "418"
       external-dns.alpha.kubernetes.io/aws-weight: "100"
-      nginx.ingress.kubernetes.io/configuration-snippet: |
```

Comment:
can be replaced by `server-snippet` - see https://github.com/ministryofjustice/hmpps-helm-charts/releases/tag/generic-service-3.0.0

Reply:
Oh nice, thanks.  Was just going to fall back to using the default role but I'll add that in.